jre/kotlin/AmpliApp/README.md

```diff
@@ -0,0 +1,30 @@
+# Ampli JRE SDK (Kotlin)
+An example app using the Ampli JRE SDK
+
+# Usage
+
+### Setup the project
+You will need to do the following before running the app.
+
+1. Create a `.env` with your API key
+    1. `cp .env.example .env`
```

Comment:
Nice, using .env here is great. Can update the Android examples to match?

Reply:
@justin-fiedler It seems it's possible in some way - https://github.com/cdimascio/dotenv-kotlin#android-usage. Should we use it? Maybe it's not simpler than the current `secrets.xml`

Reply:
Ok thanks, let's keep the current secrets.xml